superdesk/publish/formatters/ninjs_formatter.py

```diff
@@ -407,6 +407,9 @@ def _format_authors(self, article):
                 "name": user.get('display_name', author.get('name', '')),
                 "role": author.get('role', ''),
                 "biography": user.get('biography', author.get('biography', '')),
+                "facebook": user.get('facebook', author.get('facebook', '')),
```

Comment:
I think we should only add those if it's non-empty, otherwise this might break some clients even if they don't use the feature